source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -37,6 +37,18 @@ The following differences to the standard C++ range algorithms apply:
 Except for these differences, the signatures of parallel range algorithms correspond to the working draft
 of the next edition of the C++ standard (C++26).
 
+Auxiliary definitions
++++++++++++++++++++++
+
+The following auxiliary entities are only defined for the purpose of exposition, to aid the specification
+of parallel range algorithms.
+
+.. code:: cpp
+
+   // C++20 analogue of std::projected_value_t; exposition only
+   template<typename I, typename Proj>
+   using projected-value-type = std::remove_cvref_t<std::invoke_result_t<Proj&, std::iter_value_t<I>&>>;
```

Comment:
I understand that it's perhaps hard to make exposition only stuff italic in RST but how about the following?

```suggestion
   using /* projected-value-type */ = std::remove_cvref_t<std::invoke_result_t<Proj&, std::iter_value_t<I>&>>;
```

Reply:
Applied.